src/screens/signup/signingUpPage1.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { ScrollView, TouchableWithoutFeedback, Keyboard } from 'react-native';
+import {
+  Container,
+  Welcome,
+  DefaultText,
+  CustomText,
+  EmailBox,
+  NameBox,
+  width,
+  height
+} from './signingUpPage1Styles';
+import Input from '../../components/input';
+import Button from '../../components/button';
+import { theme } from '../../theme/types';
+
+interface SignUpPage1Props {
+  testID?: string;
+}
+
+export default function SignUpPage1(props: SignUpPage1Props) {
+  return (
+    <ScrollView>
+      <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
+        <Container>
+          <Welcome>
+            <DefaultText>Good to have you,</DefaultText>
+            <CustomText>Beautiful Woman</CustomText>
+          </Welcome>
+          <EmailBox>
+            <Input
+              placeholder="clairetamara08@gmail.com"
```

Comment:
```suggestion
              placeholder="Enter email"
```
I understand you are trying to replicate what you saw on the UI, but this is not part of what to replicate. Please change it to the suggestion I made